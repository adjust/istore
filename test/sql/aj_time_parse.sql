SELECT '2013-12-31'::aj_time;
SELECT '2013-12-31 23:00:00.000000'::aj_time;