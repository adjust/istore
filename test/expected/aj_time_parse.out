SELECT '2013-12-31'::aj_time;
       aj_time       
---------------------
 2013-12-31 00:00:00
(1 row)

SELECT '2013-12-31 23:00:00.000000'::aj_time;
       aj_time       
---------------------
 2013-12-31 23:00:00
(1 row)

